site/content/docs/multistep-checks/_index.md

```diff
@@ -29,6 +31,8 @@ Examples of API sequences might be:
 
 Monitoring your API user flows instead of individual endpoints gives confidence that your product as a whole works as intended and that the expected interactions between API calls are functional.
 
+![Multistep check overview page](/docs/images/multistep-api-checks/multistep-check-overview.png)
+
 Multistep checks are powered by `@playwright/test`'s [API Testing](https://playwright.dev/docs/api-testing) mode. Meaning you get all of the power of our typical API checks in the form of a programmable `@playwright/test` check.
 
 > Multistep checks are only supported on runtime 2023.09 or later. See [Runtimes](/docs/runtimes/) for more details.
```

Comment:
@guolau on the part just following, where it is said that

> The following code is a valid Multistep check

Can we adjust this? What is shown is not a multistep check, but rather the PWT spec it is built on. I think it's important to make this clearer (also in the "Breaking down a multistep check" section) as right after we actually assume that distinction when talking about "Creating multistep checks"